pkg/plugin/driver/daemonset/daemonset.go

```diff
@@ -218,10 +252,14 @@ func (p *Plugin) Monitor(kubeclient kubernetes.Interface, availableNodes []v1.No
 }
 
 func (p *Plugin) GetSessionID() string {
-	return p.DfnTemplateData.SessionID
+	return p.SessionID
 }
 
 // GetName returns the name of this DaemonSet plugin
 func (p *Plugin) GetName() string {
 	return p.Definition.Name
 }
+
+func getMasterAddress(hostname string) string {
+	return fmt.Sprintf("http://%s/api/v1/results/global", hostname)
```

Comment:
http?  not https?

Reply:
not yet! This is foundational work for the HTTPS work.